_includes/manuals/1.8/5.5.1_backup.md

```diff
@@ -4,24 +4,12 @@ instance.
 
 ### Database
 
-#### PostgreSQL
+#### PostgreSQL and MySQL
 
-The `pg_dump` command can be used to backup the contents of a PostgreSQL
-database to a text file. The most reliable way of backing up PostgreSQL
-database is:
+Run ```foreman-rake db:dump```. It will print a message when it finishes with the dump
+file location relative to the Foreman root.
 
-    su postgres -c "pg_dump -Fc foreman > foreman.dump"
-
-#### MySQL database
-
-The `mysqldump` command can be used to backup the contents of your MySQL
-database to a text file.
-
-    mysqldump --opt foreman > foreman.sql
-
-Please follow MySQL documentation about other options for doing backups.
-
-#### SQLite database
+#### SQLite
 
 To backup SQLite database perform this simple step:
```

Comment:
Do you think we should recommend the rake task for sqlite too, or just leave as-is?